src/workload.py

```diff
@@ -165,3 +166,13 @@ def get_service_pid(self) -> int:
                     return int(pid)
 
         raise snap.SnapError(f"Snap {self.SNAP_NAME} pid not found")
+
+    @override
+    def get_version(self) -> str:
+        if not self.kafka.present:
+            return ""
+        try:
+            version = re.split(r"[\s\-]", self.run_bin_command("topics", ["--version"]))[0]
+        except Exception:
```

Comment:
**todo:** I can see why we're catching any error here, but I think we can just `try: except:` if we're catching every error.